src/server/server_family.cc

```diff
@@ -267,7 +267,37 @@ error_code ServerFamily::LoadRdb(const std::string& rdb_file) {
 }
 
 void ServerFamily::ConfigureMetrics(util::HttpListenerBase* http_base) {
+  // The naming should be compatible with redis_exporter, see https://github.com/oliver006/redis_exporter/blob/master/exporter/exporter.go#L111
+  // TODO: add metrics' description
+  // TODO: support other types of metrics, not just gague :)
+  auto append_metric = [](http::StringResponse* resp, absl::AlphaNum name, absl::AlphaNum val) {
```

Comment:
it's a free function, why not pull it outside into anonymous namespace (line 142) ?

Reply:
Also call it AppendMetric